api/apps/geoprocessing/test/integration/marxan-run/blm-calibration-run.e2e-spec.ts

```diff
@@ -63,19 +65,20 @@ describe(`given input data is available`, () => {
   test(
     `marxan run during binary execution`,
     async () => {
-      await fixtures.GivenMarxanIsRunning();
-      // await fixtures.ThenOutputScenarioPuDataWasPersisted();
+      await fixtures.GivenBLMCalibrationIsRunning();
+      await fixtures.ThenBlmFinalResultsArePersisted();
+      await fixtures.ThenBlmPartialResultsHaveBeenDeleted();
     },
     60000 * 15,
   );
 
-  test.skip(`cancelling marxan run`, async (done) => {
+  test(`cancelling BLM calibration run`, async (done) => {
     expect.assertions(1);
 
     fixtures
-      .GivenMarxanIsRunning()
+      .GivenBLMCalibrationIsRunning()
       .then(() => {
-        done(`Shouldn't finish Marxan run.`);
+        done(`Shouldn't finish BLM calibration run.`);
```

Comment:
If I understand correctly, this done() is called once the BLM run is completed, right? If the goal of this test is to validate that we can abort it before it ends, shouldn't this throw an exception?

Reply:
That done is merely informative, the expect for the abort it's on the catch

Reply:
yeah, I know. but the thing is: as part of this test, we expect the task to never complete, because it's aborted, right? if that's the case, shouldn't the test fail in case the task completes, as that would mean that the cancellation failed?